src/mars.d

```diff
@@ -81,10 +81,12 @@ private  void usage()
     static if (TARGET_WINDOS)
     {
         const(char)* m32mscoff = "\n  -m32mscoff       generate 32 bit code and write MS-COFF object files";
+        const(char)* mscrtlib  = "\n  -mscrtlib=<name> MS C runtime library to reference from main/WinMain/DllMain";
```

Comment:
I tried to avoid that the text spans 2 lines. Maybe this is no longer a concern as the console in Windows 10 can be resized to have more than 80 characters. There is no precedent that shows the default, though (e.g. -conf, -defaultlib).